src/meili/candidate_search.cc

```diff
@@ -87,7 +87,7 @@ CandidateCollector::WithinSquaredDistance(const midgard::PointLL& location,
         } else if (dist == 0.0) {
           snapped_node = opp_edge->endnode();
         }
-        correlated.edges.emplace_back(edgeid, dist, point, sq_distance);
+        correlated.edges.emplace_back(edgeid, dist, point, sq_distance, 0.f);
```

Comment:
this method doesn't seem to be used.. should I do it properly anyways? would require some changes to the helper method meili::helpers::Project to take a shape instead of a lazy_shape

Reply:
since it's not used I don't know if heading is even important to whoever is calling this method.. doesn't sound like it judging by the name `WithinSquareDistance`

Reply:
Heading is not used for mapmatching. But 0 is certainly not correct. And this method is used but it's templates so it's hard for IDEs to tell. I feel like given this we probably should have just made it optional and defaulted to -1.sorry I should have done more looking at the calls before adding that as a potential sokution

Reply:
found that in the veeery end.. very annoying..

Reply:
jup done